internal/events/event.go

```diff
@@ -20,6 +21,8 @@ import (
 	"gorm.io/gorm"
 )
 
+const defaultPageSize int64 = 10
```

Comment:
In order to not break anything consider starting with a very big default (I think that currently we return 200 events)

Reply:
I think currently is unlimited, so I would recommend setting -1 as default. Assisted-scraper truncates at 2000 (for some unknown reason)